content/en/docs/appstore/app-services/general-purpose-ocr.md

```diff
@@ -6,39 +6,191 @@ tags: ["Document Service", "AI", "ML", "OCR", "Industrial", "Manufacturing"]
 
 ## 1 Introduction
 
-The [General Purpose Optical Character Recognition](https://marketplace.mendix.com/link/component/118392) app service can help you extract text from images or PDF documents and get output in JSON and XML formats in bulk.
+The [General Purpose Optical Character Recognition](https://marketplace.mendix.com/link/component/118392) app service can help you extract text, tables, and barcodes from images or PDF documents and get output in XML formats in bulk.
+
+### 1.1 Features
+
+* Extract data(text,table and barcode) in XML format from images and pdf documents in bulk and map data to an entity.
+* Support [Mendix SSO](/appstore/modules/mendix-sso/)
+
+### 1.2 Limitation
+
+* Currently only supports images(in JPG, JPEG, PNG, BMP, TIFF formats) and Pdf documents. 
+* Total documents size cannot exceed 20 MB.
+* ABBYY FineReader Engine will not open images larger than 32512*32512 pixels.
+
+### 1.3 Prerequisites
+
+* This app service works best with Studio Pro 8 versions starting with [8.18.15](/releasenotes/studio-pro/8.18/#81815) and 9 versions starting with [9.0](/releasenotes/studio-pro/9.0/).
 
 ## 2 Installation
 
-1. Follow the instructions in [How to Use Marketplace Content in Studio Pro](/appstore/general/app-store-content/) to import the General Purpose OCR component into your application.
+### 2.1 Obtaining the Binding Keys {#obtain-keys}
+
+The General Purpose OCR is a premium Mendix product subject to a purchase and subscription fee. To successfully use this app service in your app, first you need to start a subscription or a trial to get the binding keys.
```

Comment:
@VivekSinghRawat-1 and Luyao
always write app service in camel case i.e. "App Service"